src/tools/translate-to-fuzz.h

```diff
@@ -172,6 +173,24 @@ class TranslateToFuzzReader {
     wasm.table.segments.emplace_back(builder.makeConst(Literal(int32_t(0))));
   }
 
+  std::map<WasmType, std::vector<Name>> globalsByType;
+
+  void setupGlobals() {
+    size_t index = 0;
+    for (auto type : { i32, i64, f32, f64 }) {
+      auto num = upTo(3);
+      for (size_t i = 0; i < num; i++) {
+        auto* glob = new Global;
```

Comment:
Does the builder API support adding stuff to the module like this? If not, maybe we should improve it.

Reply:
Not yet. Yeah, worth doing, I'll look into it.